### DISCUSSION TITLE
Stepping back as maintainer

### DISCUSSION THREAD

**Discussion by he3als** [Atlas-OS/Atlas#1252] [Announcements]
Hi everyone, 

After three amazing years of working on Atlas, I have decided to step back from maintaining the project. Atlas has grown greatly, with currently 13.5k+ GitHub stars, alongside over 1.9 million downloads. I am proud of what we have accomplished, and I hope someone volunteers to take over as a primary maintainer.

For more details, you can read my full blog post here: <https://blog.he3als.xyz/posts/leaving-atlasos>

Thank you all for being a part of my journey! 💛

**Comment by WarmXR** [comment]
Damn, thank you he3als for all the work you've put into atlas os. I wish you well in your future projects. We will all miss you. ❤️

**Comment by volkayno** [comment]
let's hope someone trustworthy and capable takes over. Thank you for your service!
